ui/shared/components/panel/variants/Frequencies.jsx

```diff
@@ -70,6 +71,7 @@ const FreqSummary = React.memo((props) => {
   return (
     <div>
       {titleContainer ? titleContainer(props) : fieldTitle}
+      {isMito && (isHet ? ' heteroplasmy' : ' homoplasmy')}
```

Comment:
you need to set the config that is being passed in to have the correct `fieldTitle` or `titleContainer` for the mito populations, not add mito specific logic to the generic helper component

Reply:
Sounds good.